src/components/modals/SecurityAlertsModal.tsx

```diff
@@ -7,27 +7,18 @@ import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 import FontAwesome from 'react-native-vector-icons/FontAwesome'
 
 import s from '../../common/locales/strings'
-import {
-  Theme,
-  ThemeProps,
-  useTheme,
-  withTheme
-} from '../services/ThemeContext'
+import { Theme, useTheme } from '../services/ThemeContext'
 import { ModalFooter, ModalScrollArea, ModalTitle } from '../themed/ModalParts'
 import { ThemedModal } from '../themed/ThemedModal'
 
-interface OwnProps {
+interface Props {
   bridge: AirshipBridge<unknown>
   messages: EdgeLoginMessage[]
   selectUser: (username: string) => void
 }
-type Props = OwnProps & ThemeProps
```

Comment:
Thanks for fixing this, but when you do the rebase, be sure you apply the fix in the correct commit. If you autosquash as-is, the first commit will be wrong , and then the second commit will put it back the right way. It would be better history if the first commit just got it right to begin with.